src/js/widgets/SelectLocation/SelectLocationWidget.jsx

```diff
@@ -1,18 +1,19 @@
 import React, { Component } from 'react';
 import ReactMapboxGl, { Layer } from 'react-mapbox-gl';
-import * as MapboxGl from 'mapbox-gl';
+import { NavigationControl } from 'mapbox-gl';
+import MapboxGeocoder from '@mapbox/mapbox-gl-geocoder';
 
-// importing the geocoder didnt seem to work at first
-const MapboxGeocoder = require('@mapbox/mapbox-gl-geocoder');
+const HERE_APP_ID = 'R3EtGwWQmTKG5eVeyLV8';
+const HERE_APP_CODE = '8aDkNeOzfxGFkOKm9fER0A';
+const MAPBOX_TOKEN = 'pk.eyJ1IjoiY3Jvd2VhdHgiLCJhIjoiY2o1NDFvYmxkMHhkcDMycDF2a3pseDFpZiJ9.UcnizcFDleMpv5Vbv8Rngw';
+const GEOCODE_DEBUG = false;
```

Comment:
I'm getting a ton of console logs even with GEOCODER_DEBUG set to false.

Reply:
Yah, if the form was still built with the 'true' flag, that would happen.